src/components/textarea/textarea.stories.mdx

```diff
@@ -32,160 +32,96 @@ import Textarea from "carbon-react/lib/components/textarea";
 ### Default
 
 <Canvas>
-  <Story name="default">
-    {() => {
-      const [state, setState] = useState("");
-      const setValue = ({ target }) => {
-        setState(target.value);
-      };
-      return <Textarea label="Textarea" value={state} onChange={setValue} />;
-    }}
-  </Story>
+  <Story name="default" story={stories.DefaultStory} />
 </Canvas>
 
 ### Disabled
 
 <Canvas>
-  <Story name="disabled">
-    <Textarea label="Textarea" disabled />
-  </Story>
+  <Story name="disabled" story={stories.DisabledStory} />
 </Canvas>
 
 ### Label Align
 
 <Canvas>
-  <Story name="label Align">
-    {() =>
-      ["right", "left"].map((alignment) => (
-        <Textarea
-          label="Textarea"
-          labelInline
-          inputWidth={50}
-          key={alignment}
-          labelAlign={alignment}
-        />
-      ))
-    }
-  </Story>
+  <Story name="label Align" story={stories.LabelAlignStory} />
 </Canvas>
 
 ### ReadOnly
 
 <Canvas>
-  <Story name="readOnly">
-    <Textarea label="Textarea" readOnly />
-  </Story>
+  <Story name="readOnly" story={stories.readOnlyStory} />
 </Canvas>
 
 ### AutoFocus
 
 <Canvas>
-  <Story name="autoFocus" parameters={{ chromatic: { disable: true } }}>
-    <Textarea label="Textarea" autoFocus />
-  </Story>
+  <Story
+    name="autoFocus"
+    story={stories.autoFocusStory}
+    parameters={{ chromatic: { disable: true } }}
```

Comment:
from previous experience the "disable chromatic" and similar params here don't seem to work when included on the `Story` component here. You instead have to do it in the tsx file: `autoFocusStory.parameters = { chromatic: {disable: true}};`. I could be wrong, but it's safer this way.